cpp/perspective/src/cpp/view.cpp

```diff
@@ -465,6 +465,10 @@ View<CTX_T>::data_slice_to_arrow(
                 fields.push_back(::arrow::field(name, ::arrow::dictionary(::arrow::int32(), ::arrow::utf8())));
                 arr = arrow::string_col_to_dictionary_array(slice, cidx, stride, extents);
             } break;
+            case DTYPE_OBJECT: {
+                fields.push_back(::arrow::field(name, ::arrow::uint64()));
```

Comment:
This just serializes the uint64 pointers to arrow right? I assume there's no way to reconstitute the object after it goes into arrow/from an arrow (not that we want to, just curious)

Reply:
need @texodus to weigh in. Ideally if the underlying object supported a serialization protocol we could turn it into json/protobuf/arrow/whatever and reconstitute it later, but the object would need to define that.

e.g. if a given library's object supported json really well, the python object could be serialized to json using the library's functionality, transmitted across the wire by psp in an arrow (just storing the json as bytes or whatever), then lets say it transmitted to js, it could then be reconstituted by the the library's js apis into their JS object.

Reply:
Lacking a specific user need for serializing python objects as Arrows, I would make this an error in `to_arrow()`, though it will be hard to prevent as `DTYPE_PTR` does not have a Perspective API type string like `"object"` yet.

Reply:
@texodus part of this eliminates DTYPE_PTR and solely relies on DTYPE_OBJECT